.github/workflows/email.yml

```diff
@@ -30,7 +32,7 @@ jobs:
               echo "COMPARE_URL= $( echo '${{github.event.repository.html_url}}/compare/${{github.event.pull_request.base.sha}}...${{ github.event.pull_request.merge_commit_sha}}' )" >> $GITHUB_ENV      
               echo "FILES_CHANGED=$(echo '${{github.event.pull_request._links.html.href}}/files' )" >> $GITHUB_ENV
               echo $LOG
-              echo "MERGE_LOG= $(echo $LOG | sed 's/\\n/<br>/g' | sed 's/\\r/ /g' ) " >> $GITHUB_ENV
+              echo "MERGE_LOG= $(echo $LOG | sed 's/\\r\\n/<br>/g' ) " >> $GITHUB_ENV
```

Comment:
Thanks for the review Brad!

As shown in the screen shot. I did not see any difference in the way the html message is formatted with <p> and with <br><br> so went with sed 's/\\r\\n/<br>/g'.

The paragraph spacing will be fixed with the change in the PR.
Git Log:
<img width="1647" alt="Git_message" src="https://user-images.githubusercontent.com/82669529/204601395-980ab75e-b6a6-485d-ba04-2f12f5b5f764.png">

Formatted html with the change.

<img width="1428" alt="html_formatted_msg" src="https://user-images.githubusercontent.com/82669529/204601806-95ce44de-5ddf-4c3b-9b21-e73c6cd78a66.png">

Reply:
Oh sorry, I didn't read the comment stream, and only reviewed the OP and the file changes, so didn't see your previous comment.  I would be fine merging this PR as-is, but I am skeptical that it will fix the formatting issue in all HTML viewers.  I'm definitely willing to try it and see, though.

I suspect that the reason you're not seeing a difference between <br> and <p> is due to some combination of (a) the particular HTML viewer you're using, (b) your example not including any line breaks (i.e., Vass didn't format to an 80-column terminal), and (c) the screen you're viewing it on being wide enough that the two end up being equivalent.  Consider, for example, PR #20979 which mixes line breaks and paragraph breaks (really double line-breaks) in the OP, and how it appears in lines 2818-2833 in GitHub's rendering of the comment as HTML: view-source:https://github.com/chapel-lang/chapel/pull/20979 .  Given my experience with HTML, I'll be surprised if a `<br>`-only solution added sufficient whitespace, but I'm willing for us to try it as implemented in this PR and see what we find.